### DISCUSSION TITLE
Reloading archlinux forum makes it scroll a bit

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#5664] [Q&A] [answered]
I have noticed a strange behavior on the ArchLinux forum. If I go to the bottom of a thread, then I hit <kbd>R</kbd> to reaload the page, the page scross up a bit, and if I reload again it scross some more, and it keeps doing it for a while, up to when it scrolls down the whole amout it scrolled up, and then it starts over again.

[This screencast](https://ufile.io/2d27bzyx) should show what I mean. I haven't opened an issue as I'm not sure this is due to qutebrowser.

**Comment by emanuele6** [comment, marked as answer]
>  [This screencast](https://ufile.io/2d27bzyx)

Can you upload that video on another website (a website where I don't have to download a file to watch it: e.g. [imgur](https://imgur.com) or [youtube](https://youtube.com))? That website feels a little sketch.
#
Anyway, here is a link to a forum thread from which this bug can be reproduced: https://bbs.archlinux.org/viewtopic.php?id=257537
It only goes up 3 times (by not very much) though.

This "bug" doesn't occur with `firefox`, but it does occur with `chromium` (it also goes up more); so this is probably caused by something upstream.

**Reply by emanuele6** [reply]
This also doesn't occur if you use `:set backend webkit` → It's likely an upstream bug in webengine

**Reply by The-Compiler** [reply]
If you see this in Chromium as well, it's an upstream bug in Chromium. I do indeed see the same, and it's mentioned in the comments of this issue: [1041835 - Inconsistent vertical scroll after reload - chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=1041835)
